windows/CommonSettings.props

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Change this to `CommonSettings.props.example` and require users to copy to `CommonSettings.props`.

Not only it's preferred, it's consistent with the Linux build (http://caffe.berkeleyvision.org/installation.html).